sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -577,8 +577,7 @@ def process_parameter(parameter):
       def insert_extra_config_field(config_name, config_object):
         config_value = config_object.lower()
         if config_value in config_value_list:
-          self.loops_pipeline[group_name]['spec'].setdefault('extraConfigs', {})
-          self.loops_pipeline[group_name]['spec']['extraConfigs'][config_name] = config_value
+          self.loops_pipeline[group_name]['spec'][config_name] = config_value
```

Comment:
It's still defined in the `config_value_list` list, which is non-configurable...

Reply:
done